mlir/test/mlir-miopen-driver/sanity_pv_xdlops.mlir

```diff
@@ -2,7 +2,6 @@
 // Sanity test to -pv produces valid MLIR for all possible data types and
```

Comment:
@asleepzzz remove `UNSUPPORTED` clause. This test is NOT run right now.

Reply:
remove UNSUPPORTED